stdlib/3/curses/textpad.pyi

```diff
@@ -0,0 +1,11 @@
+from _curses import _CursesWindow
+from typing import Callable, Union
+
+def rectangle(win: _CursesWindow, uly: int, ulx: int, lry: int, lrx: int) -> None: ...
+
+class Textbox:
+    stripspaces: bool
+    def __init__(self, w: _CursesWindow) -> None: ...
```

Comment:
Missing the undocumented `insert_before` argument (quoting the code: `def __init__(self, win, insert_mode=False):`).

Reply:
Yeah, I was just looking at the documentation. Makes sense to look at the source code to.